raiden/transfer/state.py

```diff
@@ -1255,9 +1258,11 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
     @classmethod
     def from_dict(cls, data: typing.Dict[str, typing.Any]) -> 'TransactionExecutionStatus':
         started_optional = data.get('started_block_number')
-        started_block_number = int(started_optional) if started_optional else None
+        started_block_number = (typing.BlockNumber(int(started_optional))
+                                if started_optional else None)
         finished_optional = data.get('finished_block_number')
-        finished_block_number = int(finished_optional) if finished_optional else None
+        finished_block_number = (typing.BlockNumber(int(finished_optional))
+                                 if finished_optional else None)
```

Comment:
@pirapira you missed this closing parentheses here

Reply:
I guess the opening one too.

Reply:
Done!